packages/bierzo-wallet/src/communication/identities/index.ts

```diff
@@ -1,9 +1,10 @@
-/*global chrome*/
-import { JsonRpcRequest, parseJsonRpcResponse2, isJsonRpcErrorResponse, makeJsonRpcId } from '@iov/jsonrpc';
-import { TransactionEncoder } from '@iov/core';
 import { isPublicIdentity, PublicIdentity } from '@iov/bcp';
+import { TransactionEncoder } from '@iov/core';
 import { ethereumCodec } from '@iov/ethereum';
-import { extensionId } from '..';
+/*global chrome*/
```

Comment:
Any idea why this comment moved? Maybe the extension binds it to the import line below

Reply:
Looks like a bug in the extension.